local/tractus-x-edc/config/customer/puris-backend.properties

```diff
@@ -7,7 +7,11 @@ puris.itemstocksubmodel.apiassetid=itemstocksubmodel-api-asset
 puris.productionsubmodel.apiassetid=productionsubmodel-api-asset
 puris.demandsubmodel.apiassetid=demandsubmodel-api-asset
 puris.deliverysubmodel.apiassetid=deliverysubmodel-api-asset
-puris.frameworkagreement.credential=FrameworkAgreement.traceability
+puris.frameworkagreement.credential=puris
+puris.frameworkagreement.version=1.0
+puris.purpose.name=cx.puris.base
+puris.purpose.version=1
+logging.level.org.eclipse.tractusx.puris.backend=DEBUG
```

Comment:
I would prefer to have this set to INFO by default. Otherwise the logs are getting quite bloated

Reply:
done

Reply:
done